src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -3132,6 +3155,8 @@ namespace ReportSizingManager {
                                     CoilInHumRat = OutAirFrac * FinalSysSizing(CurSysNum).PrecoolHumRat +
                                                    (1.0 - OutAirFrac) * FinalSysSizing(CurSysNum).RetHumRatAtCoolPeak;
                                 }
+                                if (DataDesInletAirTemp > 0.0) CoilInTemp = DataDesInletAirTemp;
```

Comment:
I mean, it feels like temperature = 0.0 is being used as a flag for special operation.  Is there **any** chance that this could get to this point where the user actually entered a value of zero?

Reply:
Data* variables get initialized to 0, meaning they haven't been set yet. This is how we know they haven't been set. We could instead use -999.

Reply:
Oh, you knew that.

Reply:
Unlikely a user chooses freezing as a design condition. But what about low temp systems?

Reply:
It may be worth considering at a later time.  Zero degrees Celsius has physical meaning, so it feels like it should be avoided as a flag-value.  -99999 is generally physically meaningless for values like autosized flow and capacity, so it works nicely.  Anyway, nothing to do for now.